core/test.py

```diff
@@ -1,8 +1,37 @@
+from abc import ABCMeta
+
 from unittest import TestCase
 
+from core import app
+from user.models import User
+from blog.models import Article
 from .utils import clean_html
 
 
+class BaseTest(TestCase):
+    __metaclass__ = ABCMeta
+
+    def setUp(self):
+        self.client = app.test_client()
+
+    def tearDown(self):
+        User.objects.delete()
+        Article.objects.delete()
+
+    def login_user(self):
```

Comment:
You only want to do this for testing the `/login/` view with its process.

Otherwise you just login you user:

```python
from flask_login import login_user

login_user(self.user)
```

That's what is happening in the `/login/` view ;)